runtime/bin/tapasco-build-libs

```diff
@@ -127,10 +147,69 @@ if not clean:
         if ret:
             print('Tapasco build failed!')
             sys.exit(ret)
+
+    if args.enable_sim == '' and not path.exists("%s/.setup_lock" % sdir):
+        compile_simlibs = None
+
+        def interrupt_handler(sig, frame):
```

Comment:
Please add some comments on what is actually happening in this part of the script.